NBitcoin/BIP174/PSBTInput.cs

```diff
@@ -350,33 +350,6 @@ internal void Combine(PSBTInput other)
 
 		public bool IsFinalized() => final_script_sig != null || final_script_witness != null;
 
-		/// <summary>
```

Comment:
Removing this method has nothing to do with Miniscript. But I noticed that it is legacy code from my version of PSBT. So I removed it.